### ISSUE TITLE
No Dragoncraft Schematics in Factory and Android Labs.

### ISSUE THREAD

**Issue by emreovus33** [NFINIT-Development/Planetary#57]
I have found all other schematics in all other labs but the schematics that are available in the Android and Factory labs, are impossible to find. I visited like 6 factory labs and 8 android labs so far, had no luck at all.

Stuck at chapter two because of this.

Why :(

**Comment by br1c** [comment]
It's likely that you're just rolling bad RNG. You should continue to look. The schematics are very rare.

**Comment by emreovus33** [comment]
Yeah, *very* rare to a point where it takes you 3 days to find one of them. All other schematics were easy to find for me except for blue ones. They never appeared.

Maybe raising their spawn chances a bit, perhaps the same level as the other shematics, could be better?

**Comment by br1c** [comment]
The spawn rates are appropriate; I think you just ended with a world seed that treated you very unfortunately.

**Comment by kphoenix137** [comment]
Schematics are a rarer loot, but very reasonable to find on average for most players. They are not bugged and will spawn in chests, and it is up to RNG on how long it takes you to find them.

**Comment by emreovus33** [comment]
A new update, the same thing happens with Venus dungeons too, really hard to find Kepler 22b paper.

Another thing to note is I play with a friend on a LAN server. The weird thing is when we get tired of researching labs I send him a copy of the save and when he plays alone he finds the necessary paper/schematic immediately. The issued ones for us were, again:

- All DragonCraft schematics
- Kepler 22b research paper

It is probably just a bad RNG and bad seed, but it just weirds me out how my friend can find the necessary thing when he plays the same world alone, but we can't when we play together on LAN. ._.

**Comment by kphoenix137** [comment]
I agree that you shouldn't have to spend an excess amount of time exploring the same dungeon over and over again for a specific item. I will look into increased the spawn rate slightly for the next update.

**Comment by knoxhack** [comment]
@KPhoenixX you still around? 🙂

**Comment by emreovus33** [comment]
Lol, I hope he is alive.
